repository/freeBoardCommentsRepository.js

```diff
@@ -0,0 +1,65 @@
+const connection = require("../db/connection");
+const query = require("../db/query.json");
+
+async function insertFreeBoardComments(token, free_board_id, body) {
+    try {
+        const conn = await connection();
+        const [row] = await conn.execute(query.freeBoardComments.insert, [token.id, free_board_id, body.content]);
+
+        return row;
+    } catch(error) {
+        console.log(error);
```

Comment:
이렇게 끝내면 발생한 에러를 전달하지 않고 함수가 끝납니다.
```suggestion
        console.log(error);
        throw error;
```

`throw`를 해야 자기를 호출한 함수에 오류가 발생한 내역을 전달합니다. 
그제서야 이 함수를 호출한 try블록 안에서 `catch`로 다시 전달받은 오류를 전달합니다.

나머지 try/catch 도 전부 마찬가지 이며, 그렇다고 부분별하게 던지면 안됩니다.
기능의 끝나는 지점, 오류를 전달해야하는 상황, 전체 로직에서의 중단지점을 전부 이해하고 판단해야 합니다.